apps/api/src/modules/webhook/webhook.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Post, Body, Param, HttpException, HttpStatus } from '@nestjs/common';
+import { WebhookService } from './webhook.service';
+
+@Controller('providers')
```

Comment:
We should use GraphQL not REST API to add provider.
Ideally should be part of adding Provider API but this is fine too as it gives easy control when someone wants to add or remove webhooks after adding provider.

Reply:
updated to use resolver instead of controller